tests/import_test.go

```diff
@@ -223,3 +224,63 @@ func startPrometheusPortForward(f *framework.Framework) (string, *exec.Cmd, erro
 
 	return url, cmd, nil
 }
+
+var _ = Describe("Importer Test Suite-Block_device", func() {
+	f := framework.NewFrameworkOrDie(namespacePrefix)
+	var pv *v1.PersistentVolume
+	var storageClass *k8sv1.StorageClass
+	var pod *v1.Pod
+	var err error
+
+	BeforeEach(func() {
+		pod, err = utils.FindPodByPrefix(f.K8sClient, "cdi", "cdi-block-device", "kubevirt.io=cdi-block-device")
```

Comment:
I see this whole section a couple of times in the code, so in order not to repeat yourself, I would make this a helper function in pv.go, and call it from all the places you have it copied to.

Reply:
same